chunky/src/java/se/llbit/chunky/renderer/PreviewRenderer.java

```diff
@@ -54,7 +54,7 @@ public String getDescription() {
   @Override
   public void render(DefaultRenderManager manager) throws InterruptedException {
     TaskTracker.Task task = manager.getRenderTask();
-    task.update("Preview", 2, 0, "");
+    task.update("Displaying preview", 2, 0, "");
```

Comment:
It's not the displaying that takes a bit of time, it's rendering the preview:

```suggestion
    task.update("Rendering preview", 2, 0, "");
```

Reply:
I initially chose "Rendering preview" as well, but I chose to use "Displaying preview" to avoid confusion with the actual rendering process, especially for new users. Maybe a different wording could be used?

Reply:
Preparing preview?